packages/eslint-plugin-pf-codemods/index.js

```diff
@@ -1,4 +1,7 @@
-const createListOfRules = (version) => {
+// if you want your rule to only run when explicitly called for using the --only flag, add the rule name to the below array
```

Comment:
Would be worth adding some verbiage to the contributing MD about this new array, as well as when you'd want to add to it.